### PR TITLE
Create Rule “avoid-unnecessary-css-classes/rule”

### PR DESCRIPTION
**Pull request by Geordie88**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by bradystroud — DISMISSED** [review]

**Comment by tiagov8** [comment]
Hey @Geordie88 

I think I asked you to add me as the reviewer of this one... 

I have dismissed Brady approval since there are changes to be made

File: rules/avoid-unnecessary-css-classes/rule.md
Comment (tiagov8) [inline rules/avoid-unnecessary-css-classes/rule.md:3]: 1. We don't capitalize rules' titles, please fix
2. Missing "?" a the end

File: rules/avoid-unnecessary-css-classes/rule.md
Comment (tiagov8) [inline rules/avoid-unnecessary-css-classes/rule.md:21]: 1. Please fix grammar

File: rules/avoid-unnecessary-css-classes/rule.md
Comment (tiagov8) [inline rules/avoid-unnecessary-css-classes/rule.md:25]: 1. Change from `arrow next` to `"arrow-next"` 
2. Check for other occurances

File: rules/avoid-unnecessary-css-classes/rule.md
Comment (tiagov8) [inline rules/avoid-unnecessary-css-classes/rule.md:16]: 1. Remove extra space at the beginning

File: rules/avoid-unnecessary-css-classes/rule.md
Comment (tiagov8) [inline rules/avoid-unnecessary-css-classes/rule.md:21]: Hey @Geordie88 

1. Fix grammar mistake
2. Put all classes name in quotes, so the class name is not confused with the rest of the text   
   e.g. The "View-All-Link" class is... the "arrow-text" class is...

File: rules/avoid-unnecessary-css-classes/rule.md
Comment (tiagov8) [inline rules/avoid-unnecessary-css-classes/rule.md:21]: 1. Fix "unecessary"

**Review by tiagov8 — CHANGES_REQUESTED** [review]
Hey @Geordie88 

as per https://youtu.be/mdoxkF0s0mo your good example is not correct.

1. Please fix by explaining you should reuse classes instead of creating new ones
**Tip:** Instead of images, you can use code in Markdown (easier to maintain)

PS. If you go further, the "arrow-next" class shouldn't have been created by whoever it was. So don't use it as an example

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**
